practice-app/back-end/src/api/routes/events.js

```diff
@@ -13,4 +14,15 @@ eventsRoute.get('/search', async (req, res) => {
   res.status(200).send(conferences);
 });
 
+eventsRoute.get('/filter', async (req, res) => {
+  const { radius,place } = req.query;
+
+  const location = await getLocation(place);
+  // Get 30 events within given place around radius.
+  const conferences = await filterConferences(radius, location, 30);
```

Comment:
You can also get number of events as query parameter, with a default number.

Reply:
You're right. It could be better. I will handle it soon.